pkg/build/strategies/sti/sti.go

```diff
@@ -598,13 +602,11 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 			if config.Quiet {
 				continue
 			}
-			// The log level > 3 forces to use glog instead of printing to stdout
-			if glog.V(3) {
-				glog.Info(text)
-				continue
-			}
-			fmt.Fprintf(os.Stdout, "%s\n", strings.TrimSpace(text))
+			glog.Info(strings.TrimSpace(text))
```

Comment:
well, either way you want to implement it i suppose.  but if Info, Warning, etc are going to do the delegation, then it really needs to be done in the writeln implementation, so that each of those methods don't have to repeat the delegation.